### ISSUE TITLE
errors when compiling lkm

### ISSUE THREAD

**Issue by 8bitkitkat** [darlinghq/darling#615]
I am unable to compile 'lkm', I get make errors.
The main program compiled fine.

Here is the Pastebin of the [output](https://pastebin.com/eMcDErK2).

**Comment by ahyattdev** [comment]
This is unfortunate. Could you share what version of Manjaro you are using so that we can reproduce this on our end?

**Comment by 8bitkitkat** [comment]
The kernel version is: `5.4.6-2-MANJARO` and the exact iso I used was `manjaro-kde-18.1.4-191210-linux54`. I don't know if there are any other version numbers you would need

**Comment by ahyattdev** [comment]
A common issue that will give the output you posted occurs when the running kernel is not installed anymore because it was upgraded and replaced.

Could you reboot and see if you still have this issue?

**Comment by ahyattdev** [comment]
If that turns out to be the issue and you are able to start building it, you will encounter this unrelated issue afterwards:

https://github.com/darlinghq/darling/issues/617

**Comment by 8bitkitkat** [comment]
I rebooted, updated the repo, and still the exact same result. Not even the other issue you mentioned.

**Comment by ahyattdev** [comment]
Is `linux54-headers` installed?

**Comment by 8bitkitkat** [comment]
I in/reinstalled `linux54-headers`, rebooted, attempted the build again and got the error you mentioned.

**Comment by ahyattdev** [comment]
Glad to hear that got you further. I updated the [build instructions](https://wiki.darlinghq.org/build_instructions) page on the wiki with steps to solve this issue in the future.

Too bad you're stuck at the same spot now. Closing this since we have #617 to track the LKM build issue.

It may take a while to fix, so I suggest using a different distro in a VM if you want to use Darling in the meantime.
